products/BellHybrid/apps/application-bell-main/presenters/HomeScreenPresenter.cpp

```diff
@@ -155,4 +157,16 @@ namespace app::home_screen
             isStartup = false;
         }
     }
+
+    void HomeScreenPresenter::handleCyclicDeepRefresh()
+    {
+        constexpr auto oncePerRefreshCount = 30;
```

Comment:
Consider changing naming, e.g.
```suggestion
        constexpr auto deepRefreshPeriod = 30;
```